### PR TITLE
Swagger spec for Azure SQL restore

### PR DESCRIPTION
**Pull request by pixia**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [X] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [X] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X] My spec meets the review criteria:
  - [X] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [X] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by jaredmoo** [comment]
@nathannfan FYI

We will likely have several validation warnings, I will look at these.

**Review by sergey-shandar — CHANGES_REQUESTED** [review]
There are syntax errors in this PS https://travis-ci.org/Azure/azure-rest-api-specs/jobs/238944872. @pixie could you fix them?
```
  2) External file or url references ("$ref") in a swagger spec: /home/travis/build/Azure/azure-rest-api-specs/arm-sql/2014-04-01/swagger/backupLongTermRetentionPolicies.json should be completely resolvable.:
     Uncaught Error: /home/travis/build/Azure/azure-rest-api-specs/arm-sql/2014-04-01/swagger/backupLongTermRetentionPolicies.json has references that cannot be resolved. They are as follows: 
'Error parsing /home/travis/build/Azure/azure-rest-api-specs/arm-sql/2014-04-01/swagger/backupLongTermRetentionPolicies.json \nmissed comma between flow collection entries at line 186, column 21:\n                        "enum": [\n                        ^'
      at test/syntax.js:142:17
      at node_modules/call-me-maybe/index.js:13:28
      at _combinedTickCallback (internal/process/next_tick.js:73:7)
      at process._tickCallback (internal/process/next_tick.js:104:9)

  3) External file or url references ("$ref") in a swagger spec: /home/travis/build/Azure/azure-rest-api-specs/arm-sql/2014-04-01/swagger/sql.core.json should be completely resolvable.:
     Uncaught Error: /home/travis/build/Azure/azure-rest-api-specs/arm-sql/2014-04-01/swagger/sql.core.json has references that cannot be resolved. They are as follows: 
'Error opening file "/home/travis/build/Azure/azure-rest-api-specs/arm-sql/2014-04-01/examples/DatabaseCreateLongTermRetentionRestore.json" \nENOENT: no such file or directory, open \'/home/travis/build/Azure/azure-rest-api-specs/arm-sql/2014-04-01/examples/DatabaseCreateLongTermRetentionRestore.json\''
```

**Comment by sergey-shandar** [comment]
@pixie Could resolve merge conflicts?

**Comment by sergey-shandar** [comment]
I'm still trying to find out why build failing https://travis-ci.org/Azure/azure-rest-api-specs/builds/240606820?utm_source=github_status&utm_medium=notification

**Review by sergey-shandar — APPROVED** [review]

**Comment by jaredmoo** [comment]
Let me review again before merge. Thx

File: arm-sql/2014-04-01/swagger/backups.json [hunk 1]
```diff
@@ -1,233 +1,533 @@
...
+                    "description": "The name of the database"
+                },
+                "edition": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The edition of the database"
+                },
+                "maxSizeBytes": {
+                    "readOnly": true,
+                    "type": "integer",
```
Comment (jaredmoo) [inline arm-sql/2014-04-01/swagger/backups.json]: This should be "string" instead. (And remove format int64 below).

**Review by jaredmoo — CHANGES_REQUESTED** [review]
Just 1 error to fix.

**Review by sergey-shandar — APPROVED** [review]

**Review by jaredmoo — APPROVED** [review]

**Pull request merged**